airport/airport-pojo-bl/src/main/java/berlin/tablecodes/capex/Capex.java

```diff
@@ -0,0 +1,61 @@
+package berlin.tablecodes.capex;
+
+import ua.com.fielden.platform.entity.AbstractPersistentEntity;
+import ua.com.fielden.platform.entity.DynamicEntityKey;
+import ua.com.fielden.platform.entity.annotation.CompanionObject;
+import ua.com.fielden.platform.entity.annotation.CompositeKeyMember;
+import ua.com.fielden.platform.entity.annotation.DescRequired;
+import ua.com.fielden.platform.entity.annotation.DescTitle;
+import ua.com.fielden.platform.entity.annotation.DisplayDescription;
+import ua.com.fielden.platform.entity.annotation.IsProperty;
+import ua.com.fielden.platform.entity.annotation.KeyTitle;
+import ua.com.fielden.platform.entity.annotation.KeyType;
+import ua.com.fielden.platform.entity.annotation.MapEntityTo;
+import ua.com.fielden.platform.entity.annotation.MapTo;
+import ua.com.fielden.platform.entity.annotation.Observable;
+import ua.com.fielden.platform.entity.annotation.Title;
+import ua.com.fielden.platform.reflection.TitlesDescsGetter;
+import ua.com.fielden.platform.types.Money;
+import ua.com.fielden.platform.utils.Pair;
+
+/**
+ * Master entity object.
+ *
+ * @author Developers
+ *
+ */
+@KeyType(DynamicEntityKey.class)
+@KeyTitle("Capex")
+@CompanionObject(ICapex.class)
+@MapEntityTo
+@DescTitle("Description")
+@DisplayDescription
+@DescRequired
+public class Capex extends AbstractPersistentEntity<DynamicEntityKey> {
+
+    private static final Pair<String, String> entityTitleAndDesc = TitlesDescsGetter.getEntityTitleAndDesc(Capex.class);
+    public static final String ENTITY_TITLE = entityTitleAndDesc.getKey();
+    public static final String ENTITY_DESC = entityTitleAndDesc.getValue();
+    
+    @IsProperty
+    @MapTo
+    @Title(value = "Capex", desc = "Desc for a Capex")
+    @CompositeKeyMember(1)
+    private Money amount;
```

Comment:
Amount as the unique key to identify a capital expenditure project?

Reply:
I don't understand what are the other possible options to be used? I also do not understand what exactly is a "project" within this model?

Reply:
Think about a business key that would uniquely identify a capex project.